third-party/test/index.ts

```diff
@@ -38,6 +38,7 @@ import * as glob from "glob";
 import * as os from "os"
 // END 2018-10-05: Amazon addition.
 import * as paths from "path";
+import { MochaOptions } from "mocha";
```

Comment:
All changes to this third-party file need to be divergence commented (see examples within the file)

Reply:
i dig